packages/block-library/src/site-logo/edit.js

```diff
@@ -82,36 +83,56 @@ const SiteLogo = ( {
 		toggleSelection( true );
 	}
 
-	const img = (
-		// Disable reason: Image itself is not meant to be interactive, but
-		// should direct focus to block.
-		/* eslint-disable jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/click-events-have-key-events */
-		<a
-			href={ siteUrl }
-			className={ classes }
-			rel="home"
-			title={ title }
-			onClick={ ( event ) => event.preventDefault() }
-		>
-			<span className="custom-logo-link">
-				<img
-					className="custom-logo"
-					src={ logoUrl }
-					alt={ alt }
-					onLoad={ ( event ) => {
-						setNaturalSize(
-							pick( event.target, [
-								'naturalWidth',
-								'naturalHeight',
-							] )
-						);
-					} }
-				/>
-			</span>
-		</a>
-		/* eslint-enable jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/click-events-have-key-events */
+	let img = (
+		<span className="custom-logo-link">
```

Comment:
Also is this span really needed?

Reply:
Perhaps this was only added for backwards compatibility with [get_custom_logo](https://core.trac.wordpress.org/browser/tags/5.7.1/src/wp-includes/general-template.php#L1072) ?

Reply:
I think that yeah, it was added for backwards-compatibility... Otherwise there's no real reason for it.
Should we even be concerned with backwards-compatibility for these things though? I mean, nobody expects CSS from a classic theme to work on a block theme, that's not how it's supposed to work. So I don't really know if that span makes sense... Maybe it allows for some extra styling abilities like adding extra paddings, animations or whatever other "creative" thing themes do?

Reply:
Maybe it was added to match the editor markup with the front?

Reply:
In which case it fails right now.